stacks/values.yaml

```diff
@@ -0,0 +1,38 @@
+# Visit https://github.com/okteto/charts/tree/master/stacks#reference for YAML reference
+
+services:
+  vote:
+    public: true
+    image: okteto/vote:1
+    replicas: 2
+    ports:
+      - 80
+
+  result:
+    public: true
+    image: okteto/result:1
+    command: node server.js
+    ports:
+      - 80
+
+  worker:
+    image: okteto/worker:1
+    stop_grace_period: 60
+
+  db:
+    image: postgres:9.4
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    ports:
+      - 5432
+    volumes:
+      - /var/lib/postgresql/data
+    resources:
```

Comment:
is this a compose thing, or our own? I'd remove if it's not crucial for the sample.